apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -196,14 +196,39 @@ export const IndexTable = <T extends IBaseViewModel>({
     return (
         <div>
             <h3>{heading}</h3>
+
+            <FormControl size="small">
+                <Select
+                    value={selectedFilterProperty}
+                    onChange={(changeEvent) => {
+                        const {
+                            target: { value },
+                        } = changeEvent;
+                        setSelectedFilterProperty(value as keyof T);
+                    }}
+                >
+                    {filterableProperties.map((selectedFilterProperty: any) => (
```

Comment:
Add an extra menu item for 'search all properties' that triggers searching any of the filter properties.